extensions/transform/org.eclipse.smarthome.transform.xslt/README.md

```diff
@@ -2,21 +2,33 @@
 
 Transform input using the XML Stylesheet Language for Transformations (XSLT).
 
-It expects the transformation rule to be read from a file which is stored under the `transform` folder. 
+XSLT is a standard method to transform an XML structures from one documentt into a new document with a other structure.
+
+Teh transformation expects the rule to be read from a file which is stored under the `transform` folder. 
 To organize the various transformations one should use subfolders.
 
-## Example
+General transformation rule summary:
+
+* The directive `xsl:output` defines how the output document should be structured.
+* The directive `xsl:template` specifies matching attributes for the XML node to find. 
+* The `xsl:template` tag contains the rule which specifies what should be done.
+
+The Rule uses XPath to gather the XML node information for more information have a look at the XPath transformation.
+
+## Simple Example
 
-(from [here](https://en.wikipedia.org/wiki/Java_API_for_XML_Processing#Example))
+A simple but complete XLST transformation looks like follows, taken from [here](https://en.wikipedia.org/wiki/Java_API_for_XML_Processing#Example).
```

Comment:
"XSLT", not "XLST"
"looks like the following example, which was taken from ..."